lib/mmkv-storage.ts

```
@@ -43,10 +43,11 @@ export async function initializeSecureStorage(): Promise<void> {
       SecureKeyManager.getEncryptionKey('secure'),
     ]);
 
-    // Create storage instances with secure keys
-    storage.setEncryptionKey(defaultKey);
-    cacheStorage.setEncryptionKey(cacheKey);
-    secureStorage.setEncryptionKey(secureKey);
+    // Recreate storage instances with secure keys (MMKV doesn't support changing key at runtime)
+    // Note: In React Native, encryption key is passed at construction time
+    (storage as any) = new MMKV({ id: 'linguamate-default', encryptionKey: defaultKey });
```

Biome (2.1.2)

[error] Can't assign storage because it's a constant.

This is where the variable is defined as constant.

Unsafe fix: Replace const with let if you assign it to a new value.


(lint/correctness/noConstAssign)